apps/i18n/common/en_us.json

```diff
@@ -1024,6 +1025,7 @@
   "studentControl": "Individual student control",
   "studentLockStateInstructions": "Use the table below to control the state for each individual student.",
   "studentName": "Student Name",
+  "studentAnswer": "Student Answer",
```

Comment:
Typically we capitalize like `Student answer` and `Correct answer` - did the spec say otherwise?

Reply:
Good catch!  Thanks